admin/aioseop_module_class.php

```diff
@@ -2336,6 +2336,12 @@ function get_option_html( $args ) {
 				case 'esc_html':
 					$buf .= '<pre>' . esc_html( $value ) . "</pre>\n";
 					break;
+				case 'template':
```

Comment:
It is one line right now but that does not make it irrelevant. In view of the current code, there is no other way to add support for generic templates.

I'm also not sure how ob_start() would make memory a concern. Its been used in many places without affecting memory.

Reply:
The whole point of ob_start is to store in the memory instead of letting echo render everything. My worry was when a large log was added to the memory buffer.